test/test_helper.rb

```diff
@@ -1,11 +1,20 @@
-ENV['RAILS_ENV'] ||= 'test'
+ENV['RAILS_ENV'] ||= "test"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.